.cirrus.yml

```diff
@@ -124,3 +122,4 @@ task:
         - flutter channel $CHANNEL
         - ./script/incremental_build.sh build-examples --ipa
         - ./script/incremental_build.sh drive-examples
+        - ./script/incremental_build.sh podspecs --skip url_launcher_web --skip google_sign_in_web --skip e2e_macos --skip video_player_web --no-analyze camera
```

Comment:
nit: Would it be worth doing this before any of the other steps? That way if there's a lint error this will fail immediately instead of waiting until everything else completes.

Reply:
Or maybe I should split it into its own shard so if one fails you still get the output from the others?

Reply:
That makes sense too. I think either is fine.